### PR TITLE
adding code snippet for classification estimate

### PR DESCRIPTION
**Pull request by AnastasiaLobanova**
An addition to the examples notebook with code snippet for classification estimate point

### CODE REVIEW

File: notebooks/climatiq.ipynb [hunk 1]
```diff
@@ -335,6 +436,48 @@
...
+    "[More details about the industry classification estimate endpoint here](https://www.climatiq.io/docs#classifications) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "API_token = \"your token\" # add your Climatiq API token here\n",
```
Comment (GeeWee) [inline notebooks/climatiq.ipynb]: Maybe streamline this and call it `MY_API_KEY` here as well, like in the above cells?

File: notebooks/climatiq.ipynb [hunk 2]
```diff
@@ -335,6 +436,48 @@
     "\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Industry classification estimate\n",
+    "\n",
+    "If your data comes with an industry specification code Climatiq API allows you to query emission factors, based on their NACE or ISIC industry classification codes. At the momemnt this endpoint works only with spend-based datasets, in particular EXIOBASE and GHG Protocol. \n",
```
Comment (GeeWee) [inline notebooks/climatiq.ipynb]:
```suggestion
    "If your data comes with an industry specification code Climatiq API allows you to query emission factors, based on their NACE or ISIC industry classification codes. At the moment this endpoint works only with spend-based datasets, in particular EXIOBASE and GHG Protocol. \n",
```

File: notebooks/climatiq.ipynb [hunk 3]
```diff
@@ -335,6 +436,48 @@
...
+    "        \"parameters\": {\n",
+    "            \"money\": 3000,  # provide amount of money spend on the specific activity\n",
+    "            \"money_unit\": \"usd\" # choose currency, currently we support more than 30 currencies - please check the documentation\n",
+    "        }\n",
+    "    }\n",
+    "    response = requests.post(url,headers=headers,json=data) # this establishes connection to the API\n",
+    "    response_data = json.loads(response.text) # this line queries the data\n",
+    "\n",
+    "\n",
+    "pd.json_normalize(response_data) # this line provides a common dataframe out of the API Response\n"
```
Comment (GeeWee) [inline notebooks/climatiq.ipynb]: Suggest we mention that you can print it, or "if you're using `pandas` you can get a dataframe like this" - basically I'd like to ensure that people who might not know what `pd` stands for, or maybe even what pandas is, finds this useful too.

**Review by GeeWee — CHANGES_REQUESTED** [review]
Mostly minor things, but I think it'd be nice to introduce pandas properly

File: notebooks/climatiq.ipynb (hunk 3 above)
Comment (AnastasiaLobanova) [reply notebooks/climatiq.ipynb]: Yess!

File: notebooks/climatiq.ipynb [hunk 4]
```diff
@@ -452,19 +452,19 @@
...
     "url = \"https://beta3.api.climatiq.io/classifications/estimate\"\n",
-    "authorization_header = {\"Authorization\": \"Bearer \"+ API_token} # this is an authorization header which allows API to recognize you as a valid user\n",
+    "authorization_header = {\"Authorization\": \"Bearer \"+ MY_API_KEY} # this is an authorization header which allows API to recognize you as a valid user\n",
     "\n",
     "\n",
     " data = {\n",
     "            \"classification\": {\n",
     "            \"classification_type\": \"nace2\", # here add your classification type, at the momement it will be either nace2 or isic4\n",
-    "            \"classification_code\": \"nace_code\", # here add a code from your classification, e.g. NACE 1 - \n",
+    "            \"classification_code\": \"nace_code\", # here add a code from your classification, e.g. for NACE 01.19 you would add 01.19 not that the value has to be a string\n",
```
Comment (GeeWee) [inline notebooks/climatiq.ipynb]:
```suggestion
    "            \"classification_code\": \"nace_code\", # here add a code from your classification, e.g. for NACE 01.19 you would add 01.19 - note that the value has to be a string\n",
```

**Review by GeeWee — APPROVED** [review]
One minor suggestion, but feel free to apply and merge it in! Looking good!

**Comment by GeeWee** [comment]
Should we get this in @AnastasiaLobanova ?

**Pull request merged**